webpack/assets/javascripts/react_app/components/Bookmarks/components/__tests__/SearchModal.test.js

```diff
@@ -1,19 +1,16 @@
 import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
 
-import SearchModal from '../SearchModal';
+import SearchModal from '../SearchModal/SearchModal';
 
 const props = {
   controller: 'hosts',
-  show: true,
   url: '/api/bookmarks',
-  onHide: () => {},
+  title: 'Create Bookmark',
+  onEnter: jest.fn(),
+  setModalClosed: jest.fn(),
 };
 
 const fixtures = {
-  'should not show search modal': {
-    ...props,
-    show: false,
-  },
```

Comment:
we don't need this test anymore?

Reply:
@laviro Since `SearchModal` is now a ForemanModal, the basic show/hide functionality is tested already and doesn't need to be duplicated in every use of ForemanModal.